README.md

```diff
@@ -1,2 +1,70 @@
+
+A reference implementation for the Notary signature in Golang is provided in the [`notation-core-go`](https://github.com/notaryproject/notation-core-go) library. A convenient Golang library that interacts with OCI registries and manages the relation between a signed artifact and a signature is provided in the [`notation-go`](https://github.com/notaryproject/notation-go) library. `notation-go` provides an easy way to implement the Notary signing and verification in Golang. The convenient library is also used by the [`notation` CLI](https://github.com/notaryproject/notation) that can be used by developers and CI/CD pipelines to produce portable signatures and store them together with the signed artifacts in OCI-compliant registries. The `notation` CLI implements the Notary signing and verification and can also be used to verify signatures of artifacts stored in OCI-compliant registries.
+
+You can learn more about the Notary Project on the [notaryproject.dev](https://notaryproject.dev) website. 
+
+## Repositories
+
+Here is a list of repositories under the Notary Project organization
+
+| Repository | Description |
+| ----- | -----|
+| [.github](https://github.com/notaryproject/.github) | This repository contains the Notary Project governance and other common documents that are shared across repositories. |
+| [meeting-notes](https://github.com/notaryproject/meeting-notes) | This repository contains the archived meeting notes. |
+| [notary](https://github.com/notaryproject/notary) | This repository contains the source code for the server and the client of the TUF-based implementation. |
+| [notaryproject](https://github.com/notaryproject/notaryproject) | This repository contains the Notary Project requirements, scenarios,  specifications, and security audits. |
+| [notaryproject.dev](https://github.com/notaryproject/notaryproject.dev) | This repository contains the source code and content for the [Notary Project website](https://notaryproject.dev). | 
+| [notation](https://github.com/notaryproject/notation) | This repository contains the source code for the convenient CLI implementation of the Notary signing and verification flow. |
+| [notation-go](https://github.com/notaryproject/notation-go) | This repository contains the source code for the convenient Golang library implementation of the Notary signing and verification flow. |
+| [notation-core-go](https://github.com/notaryproject/notation-core-go) | This repository contains the source code for the Golang library implementation of the Notary signature specification and wrapping (COSE and JWS). |
+| [roadmap](https://github.com/notaryproject/roadmap) | This repository is intended for keeping track of development activities in the Notary Project. It may be retired in the future. |
```

Comment:
```suggestion
| [roadmap](https://github.com/notaryproject/roadmap) | This repository is intended for keeping track of development activities in the Notary Project. It may be retired in the future as feature request and milestones are moved to the appropriate repositories. |
```